.github/workflows/manual.yml

```diff
@@ -0,0 +1,27 @@
+name: Manual workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      filename:
+        description: 'Provide the path to the upstream merge script'
+        default: 'scripts/dev/upstream_merge/upstream_merge_and_test.py'
+        required: true
+        type: string
+
```

Comment:
Can we also pass in the workitem ID here as a parameter? That way, I don't even need to touch the source code at all to modify automation_conf.json. I can just click a button on the GitHub browser, tell it to run the upstream merge script for workitem ID <so-and-so>. Currently, I'm guess the workflow would be to create a branch with the modified automation_conf.json where we write the new workitem ID, then push it, then click the button to run the pipeline. Right?